### PR TITLE
Build tagged version of communication container

### PR DESCRIPTION
**Pull request by gregorjerse**
when commit is tagged. The tag itself is used as a docker image tag.
If the commit is not tagged build the development version of the image.

Additional commit:
- Fix errors and code cleanup
- Explicit separation of development and stable tags
- Push to ECR instead of Docker hub

### CODE REVIEW

**Review by dblenkus — APPROVED** [review]

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -43,3 +46,29 @@ jobs:
...
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push ${{ matrix.image }} image
```
Comment (dblenkus) [inline .github/workflows/docker.yml:68]: There is no `matrix` variable in this job.

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -3,6 +3,9 @@ name: Build Docker Images
 on:
   push:
     branches: master
+  tags:
+    - "[0-9]+.[0-9]+.[0-9]+*"
```
Comment (dblenkus) [inline .github/workflows/docker.yml:7]: Is this also triggered for alpha releases?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -75,24 +75,24 @@ jobs:
         with:
           file: .coverage
       
-  build:
-    runs-on: ubuntu-20.04
+  # build:
```
Comment (dblenkus) [inline .github/workflows/ci.yml:78]: Uncomment this.

File: .github/workflows/docker.yml (hunk 2 above)
Comment (acopar) [reply .github/workflows/docker.yml:7]: Yes, if we only wanted stable releases, the regex would be without the asterix.

**Pull request merged**